src/ViewModel/WorkspaceSettingsDialogViewModel.cs

```diff
@@ -71,6 +78,13 @@ public RelayCommand CopySettingsToWorkspaceCommand
 				return this.copySettingsToWorkspaceCommand ?? (this.copySettingsToWorkspaceCommand = new RelayCommand(
 					() =>
 					{
+						SettingsFactory.LoadUserCreatedSettings = false;
+						SettingsFactory.UserCreatedSettingsPath = string.Empty;
+						SettingsManager settingsManager = new ShellSettingsManager(ServiceProvider.GlobalProvider);
+						WritableSettingsStore store = settingsManager.GetWritableSettingsStore(SettingsScope.UserSettings);
+						store.CreateCollection(PersonalBoilerplateSettingsStore.CollectionPath);
+						store.SetString(PersonalBoilerplateSettingsStore.CollectionPath, BoilerplateSettingsFactory.UserBoilerPlateSettings, "");
```

Comment:
Use the "remove" API instead of setting to empty string?

Reply:
Wasn't sure what you meant by "remove". On store, i did not see anything related to remove except for DeleteCollection. Replaced "" with string.Empty.

Reply:
You are looking for DeleteProperty

Reply:
Ah thank you, I will update this in the next commit.